server.js

```diff
@@ -2989,6 +2989,39 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+camp.route(/^\/sourcegraph\/([\s\S]+)\.(svg|png|gif|jpg|json)$/,
+cache(function (data, match, sendBadge, request) {
+
+  var repo = match[1];  // eg, `prismic` or `foo/bar`.
+  var format = match[2];
+  var apiUrl = "https://sourcegraph.com/.api/repos/"+repo+"/-/shield"
+  var badgeData = getBadgeData('build', data);
+  request(apiUrl, function(err, res, buffer) {
+      console.log("err" + err)
+    if (err !== null) {
+        console.log(err);
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
+      return;
+    }
+    badgeData.text[0] = "Used by"
```

Comment:
Also, you can remove that line, since it is already taken care of by line 2998.

(Also, the left-hand-side almost always starts with a lowercase.)